### PR TITLE
Refactoring for testability and maintainability purposes

### PR DESCRIPTION
**Pull request by KarataevDmitry**
Main project's separated to several class libraries (one for layer/dependency)

### CODE REVIEW

File: Wissance.MossbauerLab.Watcher/Wissance.MossbauerLab.Watcher.sln [hunk 1]
```diff
@@ -1,11 +1,15 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.32002.261
+# Visual Studio Version 17
```
Comment (EvilLord666) [inline Wissance.MossbauerLab.Watcher/Wissance.MossbauerLab.Watcher.sln:3]: Am i able to use it with VS 2019?

Comment (KarataevDmitry) [reply Wissance.MossbauerLab.Watcher/Wissance.MossbauerLab.Watcher.sln:3]: yep

Comment (KarataevDmitry) [reply Wissance.MossbauerLab.Watcher/Wissance.MossbauerLab.Watcher.sln:3]: You can [install and use Visual Studio 2022](https://learn.microsoft.com/en-us/visualstudio/install/install-visual-studio) alongside previous versions of Visual Studio, including Visual Studio 2019, Visual Studio 2017, Visual Studio 2015, Visual Studio 2013, and Visual Studio 2012.

Comment (EvilLord666) [reply Wissance.MossbauerLab.Watcher/Wissance.MossbauerLab.Watcher.sln:3]: Nice

**Pull request merged**